src/System.Buffers/tests/ArrayPool/UnitTests.cs

```diff
@@ -429,7 +429,7 @@ public static void ReturningANonPooledBufferOfDifferentSizeToThePoolThrows()
         {
             ArrayPool<byte> pool = ArrayPool<byte>.Create(maxArrayLength: 16, maxArraysPerBucket: 1);
             byte[] buffer = pool.Rent(15);
-            Assert.Throws<ArgumentException>("array", () => pool.Return(new byte[1]));
+            Assert.Throws<ArgumentException>(null, () => pool.Return(new byte[1]));
```

Comment:
> The string is definitely in the implementation in CoreLib.

It's not, though.  Take a really close look at this line ;)
https://github.com/dotnet/coreclr/blob/74cbc9e2c42ffb01c84a68adb59a2ff35ae23333/src/mscorlib/src/System/Buffers/DefaultArrayPoolBucket.cs#L91